getting_started/choose_a_plan.adoc

```diff
@@ -57,6 +57,22 @@ then your account will be terminated and you will be banned from creating
 subscriptions.
 ====
 
+[[apply-coupons]]
+== Apply Coupons
+
+When registering, you can redeem valid, one-time-use promotional coupons, if
+applicable.
+
+You can also redeem coupons after the registration process:
+
+. From the *Active Subscriptions* page, click *Manage Subscriptions*.
+
+. On the next *Subscriptions* page, click *Manage* next to the *Coupons* heading.
+
+. On the next *Subscription Coupons* page, you will see (if applicable) coupons
+that are available for you to use and those that are already in use. To apply a
+new coupon, click *Apply New Coupon* and complete the workflow.
+
```

Comment:
Currently, the coupon index page is out of date, and has been changed. There is no applicable coupons shown for the user to apply, the user only can see the applied coupons.

Reply:
@ahardin-rh We just need to remove "that are available for you to use and those"

Reply:
@yasun1 @lancejjohnson Updated. Thanks!